openeogeotrellis/deploy/batch_job.py

```diff
@@ -239,7 +239,9 @@ def main(argv: List[str]) -> None:
     metadata_file = job_dir / argv[5]
     api_version = argv[6] if len(argv) >= 7 else None
     dependencies = _deserialize_dependencies(argv[7]) if len(argv) >= 8 else {}
-    user_id = argv[8] if len(argv) >= 9 else None
+    # TODO: do we still need proxy user id inside the batch job itself?
+    proxy_user_id = argv[8] if len(argv) >= 9 else None
```

Comment:
The proxy_user_id argument can be removed.

Reasoning:

The proxy user ID is only used to submit a Spark job (spark-submit --proxy-user ...) which only happens in two situations:
- explicit start job in the web API (POST '/jobs/<job_id>/results');
- asynchronously when SHub dumps batch process results in an S3 bucket and we can act on them.

So a batch job never submits batch jobs itself.